src/ModuleManager.py

```diff
@@ -235,7 +235,7 @@ def _load_module(self, bot: "IRCBot.Bot", definition: ModuleDefinition,
             definition.filename)
         module = importlib.util.module_from_spec(import_spec)
         sys.modules[import_name] = module
-        loader = typing.cast(importlib.abc.Loader, import_spec.loader)
+        loader = typing.cast(importlib._abc.Loader, import_spec.loader)
```

Comment:
This didn't work on my setup. The other changes helped it work on Python 3.11; but `_abc` was undefined, while `abc` worked.

Reply:
@JeDaYoshi -- what is your full release string? I have `Python 3.11.3` as the output of `python3 --version`.  Python merged the _abc update for Python a couple years ago -- you'll see the PR in the 22ed4dd34495f0b793453f7c72028726a30a84ce commit message.